_upgrade-to/index.md

```diff
@@ -8,14 +8,18 @@ redirect_from:
 
 # About the process
 
-The process of upgrading from Elasticsearch OSS (including Open Distro for Elasticsearch) to OpenSearch varies depending on your current version of Elasticsearch OSS, install type, tolerance for downtime, and cost-sensitivity. Rather than concrete steps to cover every situation, we have general guidance for the process.
+The process of upgrading from Elasticsearch OSS (including Open Distro for Elasticsearch) to OpenSearch varies depending on your current version of Elasticsearch OSS, installation type, tolerance for downtime, and cost-sensitivity. Rather than concrete steps to cover every situation, we have general guidance for the process.
```

Comment:
Should you remove the parenthetical since you address the special case of upgrading from Open Distro belo? (...including Open Distro for Elasticsearch). Otherwise you might want to handle the Open Distro info sooner?